### DISCUSSION TITLE
잘 밤에 쓸데없는 생각하기.../[번역] Tig Manual

### DISCUSSION THREAD

**Discussion by giscus** [ujuc/ujuc.github.io#66] [General]
# 잘 밤에 쓸데없는 생각하기.../[번역] Tig Manual

Tig 메뉴얼 번역 축약본

https://ujuc.github.io/2016/02/10/tig-manual/

**Comment by markyang92** [comment]
대한민국에서 tig를 제일 잘 설명하신 것 같습니다. b

**Reply by ujuc** [reply]
툴 번역만 한거라 😄 감사합니다. 👍🏼
